quipucords/api/scan/serializer.py

```diff
@@ -88,7 +88,7 @@ class DisableOptionalProductsOptionsSerializer(NotEmptySerializer):
     class Meta:
         """Metadata for serializer."""
 
-        model = ExtendedProductSearchOptions
+        model = DisableOptionalProductsOptions
```

Comment:
wow that as a miss from the original PR

Reply:
DisableOptionalProductsOptions --> DisabledOptionalProductsOptions?

Reply:
Done.  All reference updated.